### ISSUE TITLE
[BUGFIX] Upgrade Deno package to reflect Postgres changes

### ISSUE THREAD

**Issue by bramirez96** [rahmanfadhil/cotton#35]
Postgres was recently updated to fix the following bug:

```bash
error: TS2345 [ERROR]: Argument of type 'T | undefined' is not assignable to parameter of type 'T | PromiseLike<T>'.
  Type 'undefined' is not assignable to type 'T | PromiseLike<T>'.      resolve(value);
              ~~~~~
    at https://deno.land/x/postgres@v0.4.5/utils.ts:94:15
```

Simply upgrading the package from pg `v0.4.5` to `v0.4.6` should fix the issue, as it has with a few other packages :)

I'm very excited to use your library as it seems fairly robust and awesome <3 Thanks for your time!

**Comment by bramirez96** [comment]
It looks like it's fixed in this Github repo, but the change isn't reflecting in the package on Deno. I would be very grateful if you could update the package please :)

**Comment by rahmanfadhil** [comment]
Fixed in [v0.7.5](https://github.com/rahmanfadhil/cotton/releases/tag/v0.7.5)
